config/environments/development.rb

```diff
@@ -67,4 +77,12 @@
 
   # Uncomment if you wish to allow Action Cable access from any origin.
   # config.action_cable.disable_request_forgery_protection = true
+  Rack::MiniProfiler.config.auto_inject = true
+
+  # Cấu hình host cho ngrok
+  config.hosts << "afd8-104-28-249-55.ngrok-free.app"
+  config.action_mailer.default_url_options = { host: 'afd8-104-28-249-55.ngrok-free.app' }
```

Comment:
ở môi trường dev thì dùng port 3000 của local chứ em
```suggestion
  config.action_mailer.default_url_options = {host: "localhost", port: 3000}
```